Java/sieveOfEratosthenes.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class sieveOfEratosthenes {
+
+	public static void main(String[] args) {
+		
+		Scanner sc = new Scanner(System.in);
+		int num = sc.nextInt();
+		//Creating a boolean array prime[0..size]
+		boolean prime[] = new boolean[num+1];
+		//Initializing each item in prime[] with true value
+		for(int i=0;i<=num;i++) {
```

Comment:
You can work without setting them true, and actually its preferred to use it without setting values to true as it saves extra efforts.

Reply:
Sure.